src/panels/GettingStartedPanel/Tasks/FocusTask.tsx

```diff
@@ -0,0 +1,26 @@
+import { Checkbox } from '@mantine/core';
```

Comment:
This file (FocusTask.tsx), MarsTrailColorTask.tsx, NavigationTask.tsx, and `PauseTimeTask.tsx` should be moved to the Tasks folder.